src/Commands/Modules/Users/TeamCommand.cs

```diff
@@ -24,7 +24,12 @@ public partial class UserModule : BrackeysBotModule
 
             EmbedBuilder builder = GetDefaultBuilder();
 
-            if (currentTeam != targetTeam)
+            if (targetTeam == null) 
```

Comment:
These changes are implemented in https://github.com/YilianSource/brackeys-bot/commit/aedf37a8a3b4e2fd4726f4e420778b28fc172fc9.

Reply:
But they are not active in the server. Not merged?

Reply:
Yeah, not merged yet I think!